lib/rubocop/cop/bundler/insecure_protocol_source.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Bundler
+      # The source `:gemcutter`, `:rubygems` and `:rubyforge` are deprecated
+      # because HTTP requests are insecure. Please change your source to
+      # 'https://rubygems.org' if possible, or 'http://rubygems.org' if not.
+      #
+      # @example
+      #   # bad
+      #   source :gemcutter
+      #   source :rubygems
+      #   source :rubyforge
+      #
+      #   # good
+      #   source 'https://rubygems.org' # strongly recommended
+      #   source 'http://rubygems.org'
+      class InsecureProtocolSource < Cop
+        MSG = 'The source `:%s` is deprecated because HTTP requests are ' \
+              "insecure. Please change your source to 'https://rubygems.org' " \
+              "if possible, or 'http://rubygems.org' if not.".freeze
+
+        def_node_matcher :insecure_protocol_source?, <<-PATTERN
+          (send nil :source
+            (sym {:gemcutter :rubygems :rubyforge}))
+        PATTERN
+
+        def on_send(node)
+          insecure_protocol_source?(node) do
+            source = source_node(node)
```

Comment:
`send` nodes are decorated with node extensions that expose `send_node.arguments`, `send_node.first_argument` and `send_node.last_argument`. 🙂

Reply:
You could also use a captures in the node pattern to extract the parts you want, using `$`.

Reply:
Thanks for your advice. I updated with node extensions API and `$`. 5efbbe1